aws-android-sdk-iot-test/src/androidTest/java/com/amazonaws/mobileconnectors/iot/MqttManagerIntegrationTest.java

```diff
@@ -747,45 +719,46 @@ public void onFailure(Throwable exception) {
 
     @Test
     public void testWebSocketConnectionWithIamAuth() throws Exception {
-        testWebsocketConnection(AWSIotMqttManager.AuthenticationMode.IAM, endpoint);
+        testWebsocketConnection(AuthenticationMode.IAM, iotClient.getEndpointAddress());
     }
 
     @Test
     public void testWebSocketConnectionWithCustomAuth() throws Exception {
-        testWebsocketConnection(AWSIotMqttManager.AuthenticationMode.CUSTOM_AUTH, endpoint);
+        testWebsocketConnection(AuthenticationMode.CUSTOM_AUTH, iotClient.getEndpointAddress());
     }
 
     @Test
     public void testWebSocketWithConnectionUsernamePassword() throws Exception {
-        testWebsocketConnection(AWSIotMqttManager.AuthenticationMode.USERNAME_PASSWORD, betaEndpoint);
+        String betaEndpoint = iotClient.getEndpointAddress("iot:Data-Beta");
```

Comment:
Things that make you go "hrmmm." Is this a thing?

Reply:
I doubt it. I wonder if this is called from anywhere.

Reply:
It is not in the official documentation.

There are two tests right now that use this value:
1. `testWebsocketReconnectionWithUsernameAndPassword`
2. `testWebSocketWithConnectionUsernamePassword`

Neither are passing.

I'm going to try moving them to a "real" endpoint address. If that doesn't work, I'll axe them.

(For another PR.)